tvm/src/codegen/llvm/codegen_llvm.h

```diff
@@ -279,6 +280,16 @@ class CodeGenLLVM : public ExprFunctor<llvm::Value*(const Expr&)>,
   // for Return
   bool has_return_{false};
 
+  // for Assert
+  struct assert_alloc_free {
+    VarExpr buffer_var;
+    Expr dev_type;
+    Expr dev_id;
+  };
+  std::vector<assert_alloc_free> assert_alloc_mem;
+  bool save_buffer_flag{false};
+  bool assert_free{false};
```

Comment:
Let's append a "_" to each variable. Also, can we rename `assert_free`? Maybe `from_assert` or `in_assert`. For `save_buffer_flag`, let's use `assert_save_buffer`.